packages/specs/src/rpc/beefy.ts

```diff
@@ -0,0 +1,20 @@
+import { RpcModuleSpec } from '@delightfuldot/specs';
+
+export const beefy: RpcModuleSpec = {
+  getFinalizedHead: {
+    docs:
+      'Returns hash of the latest BEEFY finalized block as seen by this client.\n\n' +
+      'The latest BEEFY block might not be available if the BEEFY gadget is not running\n' +
+      'in the network or if the client is still initializing or syncing with the network.\n' +
+      'In such case an error would be returned.',
+    params: [],
+    type: 'Hash',
+  },
+  subscribeJustifications: {
+    docs: 'Returns the block most recently finalized by BEEFY, alongside its justification.',
+    params: [],
+    pubsub: ['beefy_justifications', 'beefy_subscribeJustifications', 'beefy_unsubscribeJustifications'],
+    type: 'EncodedVersionedFinalityProof',
```

Comment:
Since we support decode the encoded proof, so we should remove the `Encoded` prefix since at the end the data will be decoded to plain raw values.